site/src/components/Dashboard/Navbar/Navbar.tsx

```diff
@@ -9,15 +9,14 @@ import { useProxy } from "contexts/ProxyContext";
 
 export const Navbar: FC = () => {
   const { appearance, buildInfo } = useDashboard();
-  const [_, authSend] = useAuth();
+  const { signOut } = useAuth();
   const me = useMe();
```

Comment:
Yes, because the `user` can be undefined, and the `useMe` adds one tiny "undefined check" on top of it but we probably can do better. I didn't remove it to avoid touching too many places on this PR since the goal was to remove the XState machine keeping the previous "compatibility".